IntegrationTests/GoldMaster/Texture.podspec.json.goldmaster

```diff
@@ -234,9 +234,7 @@ objc_library(
 acknowledged_target(
   name = "PINRemoteImage_acknowledgement",
   deps = [
-    "//Vendor/PINRemoteImage:PINCache_acknowledgement",
-    ":Core_acknowledgement",
```

Comment:
In the case that a user depended on a top level target, say for example `PINRemoteImage`, but didn't explicitly depend on `PINCache`, would the license of `PINCache` still be propagated with these lines removed? I thought this is how licenses were trans propagated.

Reply:
@rahul-malik - did you address this issue in the latest revision? From my understanding, the trans licenses will not propagate anymore.